workflow/react-project/src/store/randomCocktail/selectors.jsx

```diff
@@ -0,0 +1,7 @@
+import { createSelector } from '@reduxjs/toolkit';
+
+const getState = state => state;
+
+export const selectRandomCocktailLoading = createSelector(getState, state => state.randomCocktail.loading);
+
+export const selectRandomCocktailDetails = createSelector(getState, state => state.randomCocktail.details);
```

Comment:
it's okay, but `getState` make not a lot: get state and return state, there's no sense in it
better substitute `getState` to `getRandomCocktail = state => state.randomCocktail || {}`

So in selectors you'll need just to get what you need from randomCocktail object:
`export const selectRandomCocktailLoading = createSelector(getRandomCocktail, data => data.loading);`